### PR TITLE
updates to run project while targeting various regions

### PR DESCRIPTION
**Pull request by thitu**

### CODE REVIEW

**Review by alwold — APPROVED** [review]

File: src/main/resources/templates/success.html [hunk 1]
```diff
@@ -3,15 +3,18 @@
 <head>
   <meta charset="UTF-8">
   <title>Trusona Gateway Response</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
 </head>
-<body>
+<body style="text-align:left; margin-left:10em;margin-top:2em;font-family:'Lato',sans-serif;">
 
-<dl>
+<table width="50%">
```
Comment (rtrusona) [inline src/main/resources/templates/success.html:10]: Booo, what was wrong with using `<dl>`?

Comment (thitu) [reply src/main/resources/templates/success.html:10]: it was ugly AF

Comment (rtrusona) [reply src/main/resources/templates/success.html:10]: So your solution was to slap it in a table?

Comment (rtrusona) [reply src/main/resources/templates/success.html:10]: Was there some acceptance criteria for this change?

Comment (thitu) [reply src/main/resources/templates/success.html:10]: no. this is an internal tool for testing.
it's either we keep the `dl` tag and have ugly output, or semi-decent formatting on the output

**Pull request merged**